test494940.py

```
@@ -0,0 +1,41 @@
+# Function to calculate the sum of a list of numbers
+def calculate_sum(numbers):
+    total = 0
+    for i in range(len(numbers)):  # Inefficient looping
+        total += numbers[i]
+    return total
+
+print(calculate_sum(1, 2, 3, 4))  # Incorrect argument passing
+
+# Function to read a file and print its contents
+def read_file(file_path):
+    file = open(file_path, "r")  # File handle not closed
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)